pkg/apis/pravega/v1alpha1/bookkeeper.go

```diff
@@ -102,9 +106,24 @@ func (s *BookkeeperSpec) withDefaults() (changed bool) {
 		s.AutoRecovery = &boolTrue
 	}
 
+	if s.Resources == nil {
+		changed = true
+		s.Resources = &v1.ResourceRequirements{
+			Requests: v1.ResourceList{
+				v1.ResourceCPU:    resource.MustParse("500m"),
```

Comment:
Why these values are lower than the ones in `cr-detailed.yaml`? Shall we stick to a consistent defaults?

Reply:
The default values are meant to represent the bare minimum requirements for Pravega to run. These values are intended to be use in a first contact with Pravega, functional testing, or development work. Therefore, we want to keep them low so that Pravega can be deployed to small Kubernetes clusters. Whereas the values in `cr-detailed.yaml` are just an example of how the default resources would be overridden to tune Pravega resources to a higher performance cluster.